homeworks/vladislav.buzunar_buzyaka/4-objects/objects.js

```diff
@@ -0,0 +1,85 @@
+const tamagochi = {
+    name: 'тамагочi',
+    age: 23,
+    healthLevels: 5,
+    happyLevels: 3,
+    foodLevels: 4,
+    alive: true,
```

Comment:
You never change it(
You need either change it or use a getter.
We decided to create a getter instead so please review these part of code